src/arena.rs

```diff
@@ -195,13 +195,14 @@ impl<T> Arena<T> {
     }
 
     /// Adds a new value to the arena, returning a typed handle.
-    pub fn append(&mut self, value: T) -> Handle<T> {
+    #[cfg_attr(not(feature = "span"), allow(unused_variables))]
```

Comment:
let's handle this inside the function, not outside, i.e.
```rust
#[cfg(not(feature = "span"))]
let _ = span;
```

Reply:
Donezo.